service/routes.py

```diff
@@ -377,26 +377,34 @@ class CustomerResource(Resource):
     #------------------------------------------------------------------
     # RETRIEVE AN EXISTING CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('get_customers')
+    @api.response(404, "Customer not found")
+    @api.marshal_with(customer_model)
     def get(self, customer_id):
         """
         Retrieve a single customer
         This endpoint will return a customer based on their id
         """
-        app.logger.info(f"Request information for customer with customer_id: {customer_id}")
+        app.logger.info("Request information for customer with id [%s]", customer_id)
         customer = Customer.find(customer_id)
         if not customer:
-            raise NotFound(f"Customer with id '{customer_id}' was not found.")
+            raise NotFound("Customer with id '{}' was not found.".format(customer_id))
         return customer.serialize(), status.HTTP_200_OK
 
     #------------------------------------------------------------------
     # UPDATE AN EXISTING CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('update_customers')
+    @api.response(404, 'Customer not found')
+    @api.response(400, 'The posted Customer data was not valid')
+    @api.expect(customer_model)
```

Comment:
the expected payload should be update_customer_model. everything else looks good otherwise

Reply:
Agreed. Changed to'update_customer_model'.